doc/classes/FileAccess.xml

```diff
@@ -329,8 +328,7 @@
 				[csharp]
 				public override void _Ready()
 				{
-				    var f = new File();
-				    f.Open("user://file.dat", File.ModeFlags.WriteRead);
+				    var f = new FileAccess.Open("user://file.dat", File.ModeFlags.WriteRead);
```

Comment:
In C# it needs an `using` statement since the `Close` method no longer exists. Also, you forgot to remove the `new` keyword.

```suggestion
				    using var f = FileAccess.Open("user://file.dat", FileAccess.ModeFlags.WriteRead);
```